Firebase/Auth/Source/FIRAuth.m

```diff
@@ -1058,6 +1058,13 @@ - (void)phoneNumberSignInWithRequest:(FIRVerifyPhoneNumberRequest *)request
 }
 #endif
 
+/** @fn enableAuthUILogging
+    @brief Sets the isFirebaseAuthUIRequest flag to YES on the request configuration object.
+ */
+- (void)enableAuthUILogging {
+  _requestConfiguration.isFirebaseAuthUIRequest = YES;
```

Comment:
I'd rather make it a generic API, i.e.,

@property(nonatomic, strong) NSString *additionalFrameworkMarker;

The default value would be "FirebaseCore-iOS", and Firebase AuthUI can override it to be "FirebaseUI-iOS". This leaves room for future expansions, such as logging more information.

Also, please put this method in the `#pragma mark - Public Methods` section and treat it as if a public method in terms of threading.

Reply:
Done, PTAL.